kubernetes/deployment-production.tmpl

```diff
@@ -386,6 +402,7 @@ spec:
       containers:
         - name: panoptes-production-redis
           image: redis:6.0.5
+          args: ['--save ""', '--appendonly no']
```

Comment:
Did you mean to include this here? Don't we want redis data to be persistent for Panoptes?

Reply:
wtf!? absolutely not. bad me